targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L476xG/TARGET_DISCO_L476VG/PinNames.h

```diff
@@ -142,6 +142,13 @@ typedef enum {
     JOYSTICK_UP = PA_3,
     JOYSTICK_DOWN = PA_5,
     USER_BUTTON = JOYSTICK_CENTER,
+    // Standardized button names
+    BUTTON1 = USER_BUTTON,
+    BUTTON2 = JOYSTICK_LEFT,
+    BUTTON3 = JOYSTICK_RIGH,
```

Comment:
Travis caught a typo here. This should be `BUTTON3 = JOYSTICK_RIGHT,`, not `BUTTON3 = JOYSTICK_RIGH,`. Looks like a `T` was dropped  😄